.github/workflows/android.yml

```diff
@@ -0,0 +1,32 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+        cache: gradle
+
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: ./gradlew build
+
+    - name: Unit tests
+      run: ./gradlew test
```

Comment:
Maybe this should be `./gradlew testDebugunitTest`?

Reply:
I see more suitable 
`./gradlew testDebugUnitTest`
For the rest, I see the PR as correct and I do not block its approval due to this detail.

Reply:
I understand your point. I have added `test` task instead of `testDebugUnitTest` to be more flexible if we add flavours or new build variants. For example, if we create a specific class for debug, and a different copy of this class for release, and we create different test, we need to run `testReleaseUnitTest` in addition. Amyway, we can update the yml file if we need it in the future.

Reply:
Fixed. Please resolve the conversation if everything is ok.